docs/docs/classic-ml/traditional-ml/sparkml/guide/index.mdx

```diff
@@ -86,12 +85,10 @@ predictions = spark_model.transform(test_data)
 predictions.show()
 ```
 
-**Best for:** Large-scale batch processing, existing Spark infrastructure
-
 </TabItem>
 <TabItem value="pyfunc" label="PyFunc Format">
 
-The PyFunc format enables deployment outside of Spark environments:
+Enables deployment without Spark infrastructure:
```

Comment:
> without Spark infrastructure:

this might not be accurate,
it still depends on apache/spark library, and it needs to launch a local mode spark cluster for model inference.

Reply:
Yeah I think I need to say "it doesn't need to run IN Spark to work" . I'll update :D Good call out